pkg/operator/operator.go

```diff
@@ -80,6 +80,12 @@ const (
 	KubernetesAppName    = "app"
 	CollectorAppName     = "managed-prometheus-collector"
 	RuleEvaluatorAppName = "managed-prometheus-rule-evaluator"
+
+	// The Exporter version, will be exposed as part of the user agent information.
+	ExporterVersion = "0.3.3"
```

Comment:
This should already be available to use in [`export.Version`](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/51e169c405cf3f88a7036bb778ebe718759166c2/pkg/export/export.go#L451).